doc/reference/reference_sql/sql_user_guide.rst

```diff
@@ -473,34 +476,47 @@ FALSE is less than TRUE.
 
 .. _sql_data_type_integer:
 
-INTEGER values are numbers that do not contain decimal points and are
+INTEGER values are numerics that do not contain decimal points and are
 not expressed with exponential notation. The range of possible values is
 between -2^63 and +2^64, or NULL.
 
 .. _sql_data_type_unsigned:
 
-UNSIGNED values are numbers that do not contain decimal points and are not
+UNSIGNED values are numerics that do not contain decimal points and are not
 expressed with exponential notation. The range of possible values is
 between 0 and +2^64, or NULL.
 
 .. _sql_data_type_double:
 
-DOUBLE values are numbers that do contain decimal points (for example 0.5) or
+DOUBLE values are numeric that do contain decimal points (for example 0.5) or
 are expressed with exponential notation (for example 5E-1).
 The range of possible values is the same as for the IEEE 754 floating-point
-standard, or NULL. Numbers outside the range of DOUBLE literals may be displayed
+standard, or NULL. Nuumerics outside the range of DOUBLE literals may be displayed
 as -inf or inf.
 
 .. _sql_data_type_number:
 
 NUMBER values have the same range as DOUBLE values.
 But NUMBER values may also also be integers, and, if so,
 arithmetic operation results will be exact rather than approximate.
-For example, if a NUMBER column ``X`` contains 5, then ``X / 2`` is 2, an integer.
+For example, if a NUMBER column ``X`` contains 5, then ``X / 2`` would be 2 if division is legal.
 There is no literal format for NUMBER (literals like ``1.5`` or ``1E555``
 are considered to be DOUBLEs), so use :ref:`CAST <sql_function_cast>`
-to insist that a number has data type NUMBER, but that is rarely necessary.
+to insist that a numeric has data type NUMBER, but that is rarely necessary.
 See the description of NoSQL type :ref:`'number' <index-box_number>`.
+Support for arithmetic and built-in functions with NUMBERs was removed in Tarantool version 2.10.1.
```

Comment:
Should we mention, that this is about "arithmetic" built-in functions? Some functions, such as typeof(), max(), least() and so on can work with NUMBERs.

Reply:
Okay, I changed to  built-in arithmetic functions